jina/executors/evaluators/__init__.py

```diff
@@ -0,0 +1,28 @@
+__copyright__ = "Copyright (c) 2020 Jina AI Limited. All rights reserved."
+__license__ = "Apache-2.0"
+
+from typing import Iterable
+from jina.proto import jina_pb2
+
+from .. import BaseExecutor
+
+
+class BaseEvaluator(BaseExecutor):
```

Comment:
I think you have to adapt the `register_class` function in the `ExecutorType` meta class and add `evaluate` as another `prof_funcs`

Reply:
yes yes I have to do a lot of work, I just pretend to have a starting point of discussion